internal/host/transition_test.go

```diff
@@ -2372,7 +2373,8 @@ var _ = Describe("Refresh Host", func() {
 
 				// Test setup - Cluster creation
 				machineCidr := common.TestIPv4Networking.PrimaryMachineNetworkCidr
-				cluster = hostutil.GenerateTestCluster(clusterId, machineCidr)
+				//TODO(mko) FIXME
```

Comment:
What's the issue here? TODOs should be detailed as well :)

Reply:
This is some internal placeholder, I'll remove it once all the unit tests pass :)